src/index.ts

```diff
@@ -20,6 +22,8 @@ import {
   projectPackageJson,
 } from './util/packageJson.js';
 
+const projectDependencies = projectPackageJson.dependencies || {};
```

Comment:
I actually like this pattern better than what we have, let's copy it over here too:

- https://github.com/upleveled/preflight/blob/be0499c7b5d41d9aa71114cccecac1fd521dd999/src/checks/noDependencyProblems/noDependenciesWithoutTypes.ts#L33
- https://github.com/upleveled/preflight/blob/be0499c7b5d41d9aa71114cccecac1fd521dd999/src/checks/noDependencyProblems/nextJsProjectHasSharpInstalled.ts#L8

Reply:
done in 560b36a